data/aerial-photography/index.html

```diff
@@ -77,7 +77,7 @@ <h3 id="NAIP">
       <li>False Color (CIR)
       <li>~1-meter pixels
     </ul>
-    <p><a href="https://www.fsa.usda.gov/programs-and-services/aerial-photography/imagery-programs/naip-imagery/index">National Agriculture Imagery Program</a> is color aerial photography typically collected every 2 or 3 years in the middle of the summer. Starting in 2006 the acquisitions have been statewide. The resolution is usually 1 meter with a horizontal positional accuracy of 5 meters. In addition to the 3-band RGB color imagery, there is a color-infrared dataset from 2006 and a 3-band and 4-band dataset from 2009. The 2011 dataset is 4-band for the TIFF images and 3-band for the county moasics. The 2014 dataset is 3-band with separate single-band image tiles for the 4th band for the TIFF images, and the county moasics are 3-band. Currently there is NAIP imagery from 2003, 2004, 2006, 2009, 2011, 2014, and 2016.</p>
+    <p><a href="https://www.fsa.usda.gov/programs-and-services/aerial-photography/imagery-programs/naip-imagery/index">National Agriculture Imagery Program</a> is color aerial photography typically collected every 2 or 3 years in the middle of the summer. Starting in 2006 the acquisitions have been statewide. The resolution is usually 1 meter with a horizontal positional accuracy of 5 meters. In addition to the 3-band RGB color imagery, there is a color-infrared dataset from 2006 and a 3-band and 4-band dataset from 2009. The 2011 dataset is 4-band for the TIFF images and 3-band for the county mosaics. The 2014 dataset is 3-band with separate single-band image tiles for the 4th band for the TIFF images, and the county mosaics are 3-band. Currently there is NAIP imagery from 2003, 2004, 2006, 2009, 2011, 2014, and 2016.</p>
```

Comment:
Here I'm thinking we need a "the" before "National Agriculture Imagery Program"; but I'm also wondering: the way the sentence is worded makes it read that the NAIP _is_ the photography (which isn't correct, right?). What's the NAIP's role in this? That can help us with the wording.

Reply:
NAIP is the program to collect the imagery. So NAIP is basically the product.